src/main/java/org/session4/homework/Main.java

```diff
@@ -0,0 +1,23 @@
+package org.session4.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Main {
```

Comment:
Please follow the `Coding guideline` and leave an empty row after the class declaration.

Reply:
I forgot, since in the IDE there is a space filled with my name in between.